webapp/src/app/components/dialogs/map-settings/map-content-settings/map-content-settings.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, OnInit, Input, EventEmitter, Output } from '@angular/core';
+
+@Component({
+  selector: 'app-map-content-settings',
+  templateUrl: './map-content-settings.component.html',
+  styleUrls: ['./map-content-settings.component.scss', '../map-settings.component.scss']
+})
+export class MapContentSettingsComponent implements OnInit {
+	@Input() settings;
+	@Output() onSettingsChange = new EventEmitter<any>()
+	@Input() mapSettings;
```

Comment:
this shouldn't be an input field. MapSettings are globally defined in a single json and don't change